cmd/crane/cmd/mutate.go

```diff
@@ -91,7 +96,17 @@ func NewCmdMutate(options *[]crane.Option) *cobra.Command {
 				log.Fatalf("mutating config: %v", err)
 			}
 
-			img = mutate.Annotations(img, annotations)
+			if removeMode {
+				manifest, err := img.Manifest()
+				if err != nil {
+					log.Fatalf("get manifest: %v", err)
+				}
+				for k, _ := range annotations {
+					delete(manifest.Annotations, k)
+				}
+			} else {
+				img = mutate.Annotations(img, annotations)
```

Comment:
Should we have some `mutate.RemoveAnnotations` method that we use here? I think it's confusing and inconsistent at least, and a bug at most, that `img.Manifest()` returns a mutable `manifest` that modifies the `img`. When we eventually fix that behavior, we'll want to make sure we don't break this code.